emailer.py

```diff
@@ -0,0 +1,167 @@
+import email
+import smtplib
+
+from config import (
+    CannotLoadConfiguration,
+)
+
+
+class Emailer(object):
+    """A class for sending small amounts of email."""
+
+    # Goal and setting names for the ExternalIntegration.
+    GOAL = 'email'
+    PORT = 'port'
+    FROM_ADDRESS = 'from_address'
+
+    # Constants for different types of email.
+    VALIDATION = 'validation'
+    NOTIFICATION = 'notification'
+
+    EMAIL_TYPES = [VALIDATION, NOTIFICATION]
+
+    DEFAULT_NOTIFICATION_SUBJECT = "This address designated as the %(rel)s for %(library)s"
+    DEFAULT_VALIDATION_SUBJECT = "Validate the %(rel)s for %(library)s"
```

Comment:
I think "Confirm" is more common language for these kind of emails

Reply:
and is the rel a human-readable string?

Reply:
Yes, it is. I've renamed it to `rel_desc`.